Doc/library/decimal.rst

```diff
@@ -309,12 +309,12 @@ using the :meth:`clear_flags` method. ::
    Context(prec=9, rounding=ROUND_HALF_EVEN, Emin=-999999, Emax=999999,
            capitals=1, clamp=0, flags=[Inexact, Rounded], traps=[])
 
-The *flags* entry shows that the rational approximation to :const:`Pi` was
+The *flags* entry shows that the rational approximation to ``Pi`` was
```

Comment:
This one doesn't seem quite right (but it's no more wrong than it was before), since `Pi` doesn't refer to anything in code-space. I'd be tempted to use either just `pi` in plain text (no backticks), or the Unicode `π` character. (But not `𝜋`, which is technically the _right_ character to use, but is outside the BMP and so may well show up oddly in many fonts.)

Reply:
And the uppercase `P` especially bugs me.

Not blocking for this PR, though; no worse than it was before.

Reply:
Agreed with both. Implementing both options as applyable suggestions (since I don't see any other instances here):

```suggestion
The *flags* entry shows that the rational approximation to pi was
```

or

```suggestion
The *flags* entry shows that the rational approximation to π was
```